pkg/api/handlers/modelprovider.go

```diff
@@ -116,10 +116,15 @@ func convertToolReferenceToModelProvider(ctx context.Context, gClient *gptscript
 		return types.ModelProvider{}, err
 	}
 
+	name := ref.Status.Tool.Name
```

Comment:
Looking to get the user friendly name of the model provider here, otherwise use the id equivalent (ex. `Open AI` instead of `openai-model-provider`.